Makefile

```diff
@@ -623,9 +623,13 @@ remotesystem:
 
 .PHONY: localapiv2
 localapiv2:
-	env PODMAN=./bin/podman ./test/apiv2/test-apiv2
-	env CONTAINERS_CONF=$(CURDIR)/test/apiv2/containers.conf PODMAN=./bin/podman ${PYTHON} -m unittest discover -v ./test/apiv2/python
-	env CONTAINERS_CONF=$(CURDIR)/test/apiv2/containers.conf PODMAN=./bin/podman ${PYTHON} -m unittest discover -v ./test/python/docker
+	env PODMAN=./bin/podman stdbuf -o0 -e0 ./test/apiv2/test-apiv2
+	env CONTAINERS_CONF=$(CURDIR)/test/apiv2/containers.conf PODMAN=./bin/podman \
+		pytest --disable-warnings ./test/apiv2/python
+	touch test/__init__.py
+	env CONTAINERS_CONF=$(CURDIR)/test/apiv2/containers.conf PODMAN=./bin/podman \
+		pytest --disable-warnings ./test/python/docker
+	rm -f test/__init__.py
```

Comment:
Are you at all concerned if pytest fails, the `__init__.py` file won't get removed?  If so, something like this could work:

```
	touch test/__init__.py
	if ! env CONTAINERS_CONF=$(CURDIR)/test/apiv2/containers.conf PODMAN=./bin/podman \
		pytest --disable-warnings ./test/python/docker; then \
			rm -f test/__init__.py; exit 1; fi
	rm -f test/__init__.py
```

If not, then forget it, it's not worth the extra complexity.

Reply:
@cevich Not worth it. I was thinking of just adding the zero byte file to the repo and be done with creating/removing it.

Reply:
IIRC there was an important reason it was done in the `Makefile` like this.  Something about keeping python from treating it as a package outside of this target.  Maybe I'm wrong or it's no longer relevant concern :man_shrugging:

Reply:
Not being the original author, I didn't have that history. Now that everything is under pytest (and working), I want to fold all the API tests under one directory tree rather than having the tests in two directory trees. There is some python packaging that issues that burnt Parker first time around.